modules/@angular/core/src/security/sanitization_service.ts

```diff
@@ -0,0 +1,120 @@
+import {Injectable} from '../di/decorators';
```

Comment:
This sanitization service should live under `src/platform/dom` as it is DOM specific.